py_zipkin/zipkin.py

```diff
@@ -193,6 +219,12 @@ def __init__(
         else:
             self._span_storage = storage.default_span_storage()
         self.firehose_handler = firehose_handler
+        if kind:
+            self.kind = kind
+        elif self.kind is None:
+            self.kind = Kind.LOCAL
```

Comment:
it might be clearer here if you merge this with the ` if kind is None and include:` part above

Reply:
Done